### PR TITLE
Feature/issue 137 update docs and contributing guidelines

### LINKED ISSUE #137: Breakout Changelog, Create Github Templates
These changes aren't essential but would shorten the core `README.md`. The Templates would increase the odds that new Issues and PRs filed adhere to a format that the development team would find more useful, potentially saving time and energy for everyone involved in the process.

- Breakout a new `CHANGELOG.md` from the `README.md`
- Add a basic `ISSUE_TEMPLATE` and `PULL_REQUEST_TEMPLATE` based on the existing contributions and common practices in the open source community

If these changes sound useful for the project, please let me know in this thread, and I'd be happy to make them.

### PR DESCRIPTION
**Pull request by hydrosquall**
## What does this PR do? How does it affect users?

- Created a new `CONTRIBUTING.md` and `CHANGELOG.md` in project root
- Created templates for filing PRs and issues in `.github`
- Updated `README.md` to point to new resources
- Doesn't affect library users, but improves documentation organization for future project contributors.

## Related tickets?

- Fixes #137 

## How should this be tested?

_This section does not apply to this code because no code changes were made, but I left it intact to show how Github is tightly integrated with the idea of including checkboxes in PRs so you can see how it looks from the "Pull Requests" Tab._

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [ ] Does the code compile without warnings (check shell, console)?
- [x] Do all tests pass?
- [x] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [x] If the feature saves data to a database, can you confirm the data is indeed created in the database?

### CODE REVIEW

**Comment by tbarn** [comment]
I'll check this out tomorrow. Thanks for the PR @hydrosquall!

**Comment by hydrosquall** [comment]
Great thanks! As I was reading the docs, there were two things that I noticed that could possibly fit into the scope of this general "docs" refresh

- `Scoped Keys` is in the "Todo" of the `README.rst`, but there also appear to be docs under the heading `Create Scoped Keys`. Is it possible that this Todo might be removable? (This sort of thing could possibly be alleviated in the future by using github `milestones` feature instead/in addition to a text-based todo.
- Add a badge to PyPi next to the "travis build status" badge showing the current PyPI version of `keen`

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Note: DO NOT include your credentials in ANY code examples, descriptions, or media you make public.
+
+### Steps to Reproduce
+
+1. This is the first step
+2. This is the second step, etc.
+
+Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?
+
+### Technical details:
```
Comment (tbarn) [inline .github/ISSUE_TEMPLATE.md]: What do you think about including frameworks if applicable?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Issue Summary
+
+A summary of the issue and the browser/OS environment in which it occurs.
+
+Stack traces, error messages, and/or screenshots can be very helpful here.
+
+Note: DO NOT include your credentials in ANY code examples, descriptions, or media you make public.
+
+### Steps to Reproduce
```
Comment (tbarn) [inline .github/ISSUE_TEMPLATE.md]: Maybe mention sharing code is super helpful too?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+First off, thank you for considering contributing to this official Keen IO client. It's people like you that make Keen IO such a great tool.
+
+We put these guidelines together to try and make working with our SDK as straight forward as possible, and hopefully help you understand how we communicate about potential changes and improvements.
+
+Improving documentation, bug triaging, building modules for various frameworks or writing tutorials are all examples of helpful contributions we really appreciate.
+
+Please, don't use the issue tracker for support questions. If you have a support question please come hang out in our Slack  at http://keen.chat or send an email to team@keen.io.
+
+## Guidelines
```
Comment (tbarn) [inline CONTRIBUTING.md]: I think when keen-js got updated, the newer version of this section might have gotten lost. Here's the updated version to include: https://github.com/keen/keen-js/blob/v3/CONTRIBUTING.md#guidelines

We have an awesome Keen IO Community CoC to include.

**Comment by tbarn** [comment]
I am 👍 for removing that To Do section. It's old and out of date. We have an issue open for async and scoped keys are deprecated. 

And I'm also 👍 for adding a PyPi badge. I think they stopped sharing downloads info so their old badge isn't great. I am not sure which badge you are thinking about now. Any idea?

**Comment by hydrosquall** [comment]
Hey @tbarn -

I took a pass at all the suggested changes, but feel free to let me know if you'd like further edits.

The badge I added just shows the current version number, which isn't necessarily as nice as the downloads statistics but also conveniently makes it super clear that this package is just a `pip install` away.

I also upgraded the travis badge for good measure because it looked a little blurry when it was right next to the new SVG PyPI badge.

**Comment by tbarn** [comment]
Looks awesome @hydrosquall! 🎉 

Here's the Hacktoberfest shirt form to fill it out if you haven't already: https://goo.gl/forms/dNIeqPkZmvLoziOi1
